team.html

```diff
@@ -1,333 +1,472 @@
+            </li>
+            <li>
+                Keep an overview and help coordinate all activities that have some involvement of Gammapy,
+                such as e.g. papers, presentations or posters about or using Gammapy at gamma-ray astronomy meetings or
+                conferences, or tutorials at schools / workshops on gamma-ray astronomy data analysis.
+            </li>
+            <li>
+                Maintain Gammapy communication channels (mailing lists, slack, github, ...)
+                as well as the content at <a href="http://gammapy.org/">http://gammapy.org/</a>
+            </li>
         </ul>
-<!--        
-        <br/>
-        <h3 id="Gammapyorg_web_page_maintainer">Gammapy.org web page maintainer</h3>
-        <p>Manage the <a href="http://gammapy.org">gammapy.org</a> web site,
-          including:</p>
+        <p>
+            The current project manager is <strong>Roberta Zanin</strong>, the deputy project manager is <strong>Bruno
+            Khelifi</strong>.
+        </p>
+        <p class="small">
+            The role of project manager is very time-intensive. The project manager and deputy should aim to distribute
+            tasks to
+            other contributors in the Gammapy team, and split the remaining work.
+        </p>
+
+        <h2 id="lead-developer">Lead developer</h2>
+        <p class="lead">
+            The lead developer is the technical executive lead for the Gammapy project.
+        </p>
+        <p>
+            The lead developer is appointed by the Gammapy coordination committee, and works closely with the Gammapy
+            coordination committee, project manager, core developers and community contributors.
+        </p>
+        <p>
+            Responsibilities include:
+        </p>
         <ul>
-          <li>Maintaining contributor/roles list</li>
-          <li>Managing pull requests to the website repository</li>
+            <li>
+                Organise and drive all technical aspects of the project on a day-to-day basis.
+                Keep the overview of ongoing activities, schedules and action items and follow up
+                to make sure all important things get done.
+            </li>
+            <li>
+                Propose and outline development activities for Gammapy, e.g. by writing, co-authoring
+                or reviewing "proposal for improvement of Gammapy (PIGs)" documents.
+            </li>
+            <li>
+                Serve on the Gammapy coordination committee, as the link between CC and the development team.
+                Present and explain PIGs to the CC, and make a recommendation.
+            </li>
+            <li>
+                Manage the Gammapy developer / maintainer / contributor team. Motivate and assign responsibilities to
+                other core developers to maintain sub-packages of Gammapy and to review incoming pull requests
+                or take on a mentoring role for new Gammapy contributors.
```

Comment:
Keep only "Manage the Gammapy developer / maintainer / contributor team. Assign sub package responsibilities to
 other core developers"
The part on review incoming pull request and mentoring can be put in the paragraph for Core developers

Reply:
yes, otherwise it will be difficult to understand the role of the sub-package maintainers.